### DISCUSSION TITLE
How to merge local people results and list items (Contact list) in one result web part?

### DISCUSSION THREAD

**Discussion by DominikLeitgeb** [microsoft-search/pnp-modern-search#2936] [Q&A]
We are using custom search (PnP Search v4) for local people to find users in SharePoint online.
Now we want to add a contact list to the search results.
The list entries are synchronized by a script from an on prem AD. 
To use two result web parts is  not an option, so we need another solution.
I would like to know if it is possible to implement this case somehow and how I can do that?

In my test case I created a content type for the contacts, imported contacts to the list and created a result source for SPSPeople and the content type. After a few changes in the people layout template I got all items of the contact list and all users as a result, displayed in the same way like users with the people layout template in default. But the search box doesn't work anymore for this result type.
What can I do to solve this case?

**Comment by wobba** [comment]
Can you expand on which search box and how it's not working? And the connection to what you call result type.
